ci/TriggerRedHatBuild.groovy

```diff
@@ -14,7 +14,8 @@ import groovyx.net.http.HttpBuilder
 import groovyx.net.http.HttpException
 
 if (args.size() < 3) {
-  fail('Usage: groovy TriggerRedhatBuild.groovy <version> <projectId> <apiKey>')
+  System.err.println('Usage: groovy TriggerRedhatBuild.groovy <version> <projectId> <apiKey>')
```

Comment:
why is this change required please? Couldn't immediately find a reference to this 'fail' method, but this might fit the bill better: https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#error-error-signal

Reply:
nm - I see this is no longer a Jenkins script :)

Reply:
yeah, this is a stand-alone to which jenkins delegates.

Reply:
also, I changed it because I found out that the fail() method wasn't available as I had tried to use it earlier. (bug!)